packages/core/src/checkout/checkout.ts

```diff
@@ -41,6 +42,7 @@ export default interface Checkout {
     updatedTime: string;
     payments?: CheckoutPayment[];
     channelId: number;
+    fees?: Fee[];
```

Comment:
Is this an optional field?

Reply:
Thanks for reviewing. Confirmed we will always return the field regardless of whether the experiment is enabled or disabled.